src/lib/shared/extensionController.ts

```diff
@@ -11,20 +14,22 @@ export type ExtensionData = {
   icon?: string
   url?: string
   available: boolean
+  availableInIframe: boolean
 }
 
-export const findExtensions = (onFound: (_metadata: ExtensionData) => void): void => {
+export const findExtensions = (
+  onFound: (_metadata: ExtensionData, isFrame: boolean) => void,
```

Comment:
Naming for consistency

```suggestion
  onFound: (_metadata: ExtensionData, isIframe: boolean) => void,
```